### PR TITLE
New conference talk: Next generation web applications with async PHP

### PR DESCRIPTION
**Pull request by Degra1991**
Add new conference talk: https://phpconference.com/php-core-coding/next-generation-web-applications-with-async-php/

talk_ipc

### CODE REVIEW

File: www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Next generation web applications with async PHP
+info: https://phpconference.com/php-core-coding/next-generation-web-applications-with-async-php/
+location: Berlin
```
Comment (clue) [inline www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md:4]: Should probably also include country code like other talks :+1:

File: www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+tags:
+  - next generation web applications with async php
```
Comment (clue) [inline www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md:6]: Not sure about a proper tag, but should probably be shorter and dash separated like other tags. Any thoughts?

File: www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: Next generation web applications with async PHP
+info: https://phpconference.com/php-core-coding/next-generation-web-applications-with-async-php/
+location: Berlin
+tags:
+  - next generation web applications with async php 
+  - reactphp
+  - frameworkx
```
Comment (clue) [inline www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md:8]: Okay as-is, I suppose?

**Review by clue — COMMENTED** [review]
@Degra1991 Thanks for filing this PR! Can you look into the below points?

File: www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md (hunk 1 above)
Comment (Degra1991) [reply www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md]: Added country code

File: www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md (hunk 2 above)
Comment (Degra1991) [reply www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md]: Tag shortened sensibly and dashes added

File: www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md (hunk 3 above)
Comment (Degra1991) [reply www/_talks/2021-06-10  Next generation web applications with async PHP-ipc.md]: The type of event has always been tagged so far. Accordingly, I would suggest to keep #conference.

**Review by clue — APPROVED** [review]
@Degra1991 Thanks for the update, changes LGTM! :shipit:

**Pull request merged**